README.md

```diff
@@ -1504,7 +1602,7 @@ experiment where we abort the transaction in case we observe that the values of
 
 ```ocaml
 # with_updater @@ fun () ->
-    for _ = 1 to 10_000 do
+    for _ = 1 to 100_000 do
```

Comment:
This is inherently non-deterministic and I noticed that it occasionally failed on CI.  I haven't noticed failures after increasing the number of attempts, but it can of course still happen.